### ISSUE TITLE
[Request for feature] Provide timestamp in logging

### ISSUE THREAD

**Issue by okkhoy** [MarkBind/markbind#117]
Currently, when live preview is running and file changes, the log on console only reads `info: Reload for file changed: <path>`

E.g:

```
info: Press CTRL+C to stop ...
info: Reload for file change: <path>\cs2103\website-jan18\schedule\week1\lecture.md
info: Reload for file change: <path>\cs2103\website-jan18\schedule\week1\lecture.md
```

As in the above example, we cannot say which line corresponds to when the file was  changed. It would help to have time stamp info here, to help identify the latest (or which changes went into the serving).

**Comment by damithc** [comment]
For large websites, live reloading is not very useful due to the performance problem we have now. For smaller websites, live reloading is almost instantaneous, so the timestamp does not add much value (the loaded version is guaranteed to be the latest version)

**Comment by acjh** [comment]
Resolved by #398
```
info: [14:35:25] Reload for file change: <path>\cs2103\website-jan18\schedule\week1\lecture.md
```
